### ISSUE TITLE
Add support for Redis Manager constructor in Laravel 5.7

### ISSUE THREAD

**Issue by yupmin-ct** [monospice/laravel-redis-sentinel-drivers#18]
Too few arguments to function Illuminate\Redis\RedisManager::__construct(), 2 passed in /Users/yupmin/PhpstormProjects/xxxxxxxx/vendor/monospice/laravel-redis-sentinel-drivers/src/Manager/VersionedManagerFactory.php on line 65 and exactly 3 expected

https://github.com/illuminate/redis/blob/v5.7.4/RedisManager.php
```
    /**
     * Create a new Redis manager instance.
     *
     * @param  \Illuminate\Foundation\Application  $app
     * @param  string  $driver
     * @param  array  $config
     * @return void
     */
    public function __construct($app, $driver, array $config)
    {
        $this->app = $app;
        $this->driver = $driver;
        $this->config = $config;
    }
```

https://github.com/illuminate/redis/blob/v5.6.38/RedisManager.php

```
    /**
     * Create a new Redis manager instance.
     *
     * @param  string  $driver
     * @param  array  $config
     * @return void
     */
    public function __construct($driver, array $config)
    {
        $this->driver = $driver;
        $this->config = $config;
    }
```

**Comment by yupmin-ct** [comment]
this issue is moved #19

**Comment by cyrossignol** [comment]
Support for Laravel 5.7 released in version 2.5.0.
